zkml/docs/src/llms-layers/qkv.md

```diff
@@ -0,0 +1,95 @@
+
+Then, following the [accumulation strategy](https://github.com/Lagrange-Labs/deep-prove/blob/master/docs/src/commitments.md#accumulation-of-different-polynomials) mentioned above, the prover computes the vectors:
+  
+- $\mathbf{b}_{prev} \in \mathbb{F}^{s * h}$ s.t. $\mathbf{b}_{prev}[j] = \beta(j, r_{prev})$
+- $\mathbf{b}_{new} \in \mathbb{F}^{h}$ s.t. $\mathbf{b}_{new}[j] = \beta(j, r)$
+- $\mathbf{b} \in \mathbb{F}^{(s+1)*h}= [a_1\mathbf{b}_{prev} || a_2\mathbf{b}_{new}]$
+
+Given the MLE $B$ of vector $\mathbf{b}$, and the MLEs $K$ and $V$ of the output matrices, the prover then proves the following relationship by sum-check
+\begin{equation}
+y_{agg} = \sum_{x \in \{0,1\}^{\log((s+1)h)}} B(x)(K(x) + \lambda V(x))
+\tag{2} 
+\end{equation}
+
+The sum-check proof will produce the following claims at a random point $r_s$:
+
+- Claim $B(r_s)$, which can be directly verified by the verifier
+- Claims $K(r_s)$ and $V(r_s)$, which will be later accumulated with claims $y_K$ and $y_V$, respectively, to prove that they refer to the same polynomial, using the [same poly technique](https://github.com/Lagrange-Labs/deep-prove/blob/master/docs/src/commitments.md#accumulation-for-same-polynomial). 
+
+The claims $y_k$ and $y_v$ related to the vectors $k$ and $v$ are then used to prove step 1, as shown next.
+
+#### Prove Computation of New Vectors
+A simplied variant of the sum-check employed in the general QKV layer, described in Equation (1), can be employed. The proving starts from 3 claims $y_q$, $y_k$ and $y_v$: the first one is coming directly from the output claims of the QKV layer, and it is computed over a random point $r_q \in \mathbb{F}^{\log(h)}$; the latter claims correspond to the claims $y_k$ and $y_v$ employed to prove concatenation in the same layer, and so they are evaluated over the same point $r$. 
+
+The prover computes the claims $B_q(r_q)$, $B_k(r)$, $B_v(r)$ and generates a sum-check proof for the following relationship:
+$$ y_q - B_q(r_q) + \lambda (y_k - B_k(r))  + \lambda^2 (y_v - B_v(r)) = \sum_{z \in \{0,1\}^{\log(e)}} X(z)*(W_Q(z, r_q) + \lambda W_K(z, r) + \lambda^2 W_V(z,r))
+$$
+where $X$ is the MLE of the input row $x$.
+
+#### Bind Claims With Previous Iteration
+
+Note that the claims $y_{K_{prev}}$ and $y_{V_{prev}}$, computed by the prover when proving concatentation, needs to be bound to the MLEs of the output matrices $K_{prev}$ and $V_{prev}$ computed in the previous iteration of the same QKV layer. Therefore, after the sum-check described in Equation (2), when using the same poly techique to show that claims about the output matrices $K$ and $V$ comes from the same polynomial, the prover needs to include in the same poly proof also the claims $y_{K_{prev}}$ and $y_{V_{prev}}$ computed in the sub-sequent iteration. This allows to bind the claims computed over MLEs chosen by the prover at iteration $i$ to the actual output matrices produced by the same QKV layer at iteration $i-1$.
```

Comment:
Can you show what you mean by "needs to include in the same poly proof also the claims..." more precisely ? 
Because that should be something that is shown here as well no, since we need to do it for the previous QKV layer which is described by this spec.